CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.34.0-rc.0 / 2022-02-24
+
+* [CHANGE] UI: Classic UI removed. #10208
+* [FEATURE] Tracing: OpenTelemetry based tracing. #9724, #10203, #10276
```

Comment:
I think this might fall under the `[CHANGE]` category because it removes the old Jaeger tracing support. Also, maybe that migration aspect of it should be made more clear in the description.